<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabUser">
       <attribute name="title">
        <string>Пользователи</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListView" name="listViewUsers">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QPushButton" name="pbAddUser">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pbEditUser">
              <property name="text">
               <string>Редактировать</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pbDeleteUser">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pbGetUserCard">
              <property name="text">
               <string>Выпустить карту</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTextEdit" name="teUserInfo">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableLimits"/>
          </item>
          <item>
           <widget class="QTableView" name="tableNewLimits"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pbAddLimits">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbEditLimits">
              <property name="text">
               <string>Редактировать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbDeleteLimits">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbApply">
              <property name="text">
               <string>Применить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDictionary">
       <attribute name="title">
        <string>Справочники</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabFuel">
           <attribute name="title">
            <string>Топливо</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QPushButton" name="pbAddFuel">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QTableView" name="tableFuel"/>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pbEditFuel">
              <property name="text">
               <string>Редактировать</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pbDeleteFuel">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabRez">
           <attribute name="title">
            <string>Резервуары</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0" colspan="3">
             <widget class="QTableView" name="tableTanks"/>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pbAddTanks">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pbEditTanks">
              <property name="text">
               <string>Редактировать</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pbDeleteTanks">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabCol">
           <attribute name="title">
            <string>Колонки</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="1" column="0" colspan="3">
             <widget class="QTableView" name="tablePoints"/>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pbAddPoints">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pbEditPoints">
              <property name="text">
               <string>Редактировать</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="pbDeletePoints">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Выберите номер колонки</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cbTankNum"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pbChangePointsNum">
              <property name="text">
               <string>Поменять</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabObj">
           <attribute name="title">
            <string>Объекты</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0" colspan="3">
             <widget class="QTableView" name="tableObject"/>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pbEditObj">
              <property name="text">
               <string>Редактрировать</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pbSetObj">
              <property name="text">
               <string>Настроить</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pbAddObj">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pbDeleteObj">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pbConfigure">
              <property name="text">
               <string>Конфигурировать</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="pbUserUpdate">
              <property name="text">
               <string>Обновление пользователей</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>История</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="1" column="0">
             <widget class="QPushButton" name="pbGetHistory">
              <property name="text">
               <string>Получить</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QTableView" name="tableHistory"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabConnect">
       <attribute name="title">
        <string>Подключение</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Укажите путь к базе данных:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="2" column="0" colspan="2">
              <widget class="QLineEdit" name="leUserName">
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="lblUserName">
               <property name="text">
                <string>Введите имя пользвателя:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="lblPasord">
               <property name="text">
                <string>Введите пароль:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>443</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QLineEdit" name="lePassword">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0" colspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="lePath"/>
               </item>
               <item>
                <widget class="QPushButton" name="pbView">
                 <property name="text">
                  <string>Обзор</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QCheckBox" name="cbAdminAccess">
               <property name="text">
                <string>Режим администратора</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="tristate">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QPushButton" name="pbConnect">
               <property name="text">
                <string>Подключиться</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAbout">
       <attribute name="title">
        <string>О Программе</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>25</y>
          <width>231</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Версия программы: 0.1</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actUser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Пользователи</string>
   </property>
   <property name="toolTip">
    <string>Показать таблицу пользователей</string>
   </property>
  </action>
  <action name="actDirectory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Справочники</string>
   </property>
   <property name="toolTip">
    <string>Показать справочники</string>
   </property>
  </action>
  <action name="actConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Подключение</string>
   </property>
  </action>
  <action name="actConfigurate">
   <property name="text">
    <string>Начать</string>
   </property>
  </action>
  <action name="actUpdUsers">
   <property name="text">
    <string>Пользователи</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
